### ISSUE TITLE
Could not find the drug prediction results?

### ISSUE THREAD

**Issue by Jacob-0221** [idekerlab/nest_vnn#2]
Hello,

'A deep learning model of tumor cell architecture elucidates response and resistance to CDK4/6 inhibitors' is an insightful paper. That's a wonderful and novel method to predict drug response using drug structure, DNA and the protein interactions. 

I'm trying to use your model to predict some drug response in our own cell lines. However, I met some issues-How can I get the prediction of drug response in predict.py results? I run your example files, but I get the same value in the result.txt. Is that value means the drug response?

I'm looking forward to your guidance and suggestions.

Thanks
[predict.txt](https://github.com/idekerlab/nest_vnn/files/14971419/predict.txt)

**Comment by aksinghal5590** [comment]
Hi Jacob, assuming you are using the correct input data, I've added a script "test_palbociclib.sh" in the "sample" folder to run predictions on a pre-trained palbociclib model.
The predictions should be in a predict.txt file, similar to the "sample/pretrained_model_outputs/palbociclib_1/predict.txt" file.

**Comment by Jacob-0221** [comment]
Hi Akshat, 

Thanks for your help! I have run the "test_palbociclib.sh" and got the right results.  However, I'm trying to use the train.sh file to train the model, and I did not change the code. Then I use the test.sh to get the predict file based on the trained model. However, I still got the same value in the predict.txt. I don't know why I got the same value. 

I'm looking forward to your guidance and suggestions.

Thanks
